src/main/java/com/igorbochkor/service/ManufacturerServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package com.igorbochkor.service;
+
+import com.igorbochkor.dao.ManufacturerDao;
+import com.igorbochkor.lib.Inject;
+import com.igorbochkor.lib.Service;
+import com.igorbochkor.model.Manufacturer;
+import java.util.List;
+
+@Service
+public class ManufacturerServiceImpl implements ManufacturerService {
+    @Inject
+    private ManufacturerDao manufacturerDao;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        return manufacturerDao.create(manufacturer);
+    }
+
+    @Override
+    public Manufacturer get(Long id) {
+        return manufacturerDao.get(id).orElseThrow(()
+                -> new RuntimeException("Can't find manufacturer!"));
```

Comment:
you can make the message more informative by adding id to it, or use a simpler version 
```suggestion
        return manufacturerDao.get(id).get();
```
it's up to you which option to choose

Reply:
agree